src/core/Form/TextInput/TextInput.md

```diff
@@ -101,5 +101,11 @@ import { TextInput } from 'suomifi-ui-components';
     icon="mapLocation"
     iconProps={{ fill: 'red' }}
   />
+  <TextInput
+    onBlur={(event) => console.log(event.target.value)}
```

Comment:
I would remove the `onBlur` to not distract what is done with debounce.

Reply:
I moved it to the icon example above.